src/v2/views/board/NewContactGroupBoard.test.tsx

```diff
@@ -0,0 +1,908 @@
+			getSetupServer().use(
+				rest.post('/service/soap/CreateContactRequest', async (req, res, ctx) =>
+					res(
+						ctx.status(500),
+						ctx.json({
+							Body: {
+								Fault: {
+									Reason: { Text: 'invalid request: contact must have fields' },
+									Detail: {
+										Error: {
+											Code: 'service.INVALID_REQUEST'
+										}
+									}
+								}
+							}
+						})
+					)
+				)
+			);
+			const newEmail1 = faker.internet.email();
+			const newEmail2 = faker.internet.email();
+			const newTitle = faker.string.alpha(10);
+			const { user } = setup(<NewContactGroupBoard />);
+			const contactInput = getContactInput();
+			const titleInput = screen.getByRole('textbox', { name: 'Group title*' });
+			await user.clear(titleInput);
+			await user.type(titleInput, newTitle);
+			await user.type(contactInput, newEmail1);
+			await act(async () => {
+				await user.type(contactInput, ',');
+			});
+			await user.click(screen.getByRoleWithIcon('button', { icon: ICON_REGEXP.plus }));
+			await user.type(contactInput, newEmail2);
+			await act(async () => {
+				await user.type(contactInput, ',');
+			});
+			const saveButton = screen.getByRoleWithIcon('button', {
+				name: /SAVE/i,
+				icon: ICON_REGEXP.save
+			});
+			await user.click(saveButton);
+			await screen.findByText('Something went wrong, please try again');
+			expect(screen.getByText(newTitle)).toBeVisible();
+			const memberList = await screen.findByTestId('member-list');
+			expect(within(memberList).getByText(newEmail1)).toBeVisible();
+			const chipInput = screen.getByTestId('contact-group-contact-input');
+			expect(within(chipInput).getByText(newEmail2)).toBeVisible();
+		});
+
+		it('should not use unconfirmed mails(valid chip in contactInput) in createContact request', async () => {
+			getSetupServer().use(
+				rest.post('/service/soap/CreateContactRequest', async (req, res, ctx) =>
+					res(
+						ctx.json({
+							Body: {
+								CreateContactResponse: {}
+							}
+						})
+					)
+				)
+			);
+
+			const createContactGroupSpy = jest.spyOn(client, 'createContactGroup');
+			const newEmail1 = faker.internet.email();
+			const newEmail2 = faker.internet.email();
+			const { user } = setup(<NewContactGroupBoard />);
+			const contactInput = getContactInput();
+
+			await user.type(contactInput, newEmail1);
+			await act(async () => {
+				await user.type(contactInput, ',');
+			});
+			await user.click(screen.getByRoleWithIcon('button', { icon: ICON_REGEXP.plus }));
+
+			await user.type(contactInput, newEmail2);
+			await act(async () => {
+				await user.type(contactInput, ',');
+			});
+
+			const saveButton = screen.getByRoleWithIcon('button', {
+				name: /SAVE/i,
+				icon: ICON_REGEXP.save
+			});
+			await act(async () => {
+				await user.click(saveButton);
+			});
+			await screen.findByText('Contact group successfully created');
+
+			expect(createContactGroupSpy).toBeCalledWith('New Group', [newEmail1]);
```

Comment:
I feel toHaveBeenCalled is more readable
```suggestion
			expect(createContactGroupSpy).toHaveBeenCalledWith('New Group', [newEmail1]);
```